grafana/dashboards/multicluster.json

```diff
@@ -0,0 +1,968 @@
+            "targets": [
+                {
+                    "expr": "histogram_quantile(0.5, sum(irate(response_latency_ms_bucket{dst_remote_gateway=\"$gateway\", dst_remote_gateway!=\"\", dst_remote_cluster=\"$cluster\", dst_remote_cluster!=\"\", direction=\"outbound\"}[30s])) by (le, dst_remote_gateway))",
+                    "format": "time_series",
+                    "intervalFactor": 1,
+                    "legendFormat": "p50 gateway/{{dst_remote_gateway}}",
+                    "refId": "A"
+                },
+                {
+                    "expr": "histogram_quantile(0.95, sum(irate(response_latency_ms_bucket{dst_remote_gateway=\"$gateway\", dst_remote_gateway!=\"\", dst_remote_cluster=\"$cluster\", dst_remote_cluster!=\"\", direction=\"outbound\"}[30s])) by (le, dst_remote_gateway))",
+                    "format": "time_series",
+                    "hide": false,
+                    "intervalFactor": 1,
+                    "legendFormat": "p95 gateway/{{dst_remote_gateway}}",
+                    "refId": "B"
+                },
+                {
+                    "expr": "histogram_quantile(0.99, sum(irate(response_latency_ms_bucket{dst_remote_gateway=\"$gateway\", dst_remote_gateway!=\"\", dst_remote_cluster=\"$cluster\", dst_remote_cluster!=\"\", direction=\"outbound\"}[30s])) by (le, dst_remote_gateway))",
+                    "format": "time_series",
+                    "intervalFactor": 1,
+                    "legendFormat": "p99 gateway/{{dst_remote_gateway}}",
+                    "refId": "C"
+                }
+            ],
+            "thresholds": [],
+            "timeFrom": null,
+            "timeRegions": [],
+            "timeShift": null,
+            "title": "LATENCY",
+            "tooltip": {
+                "shared": true,
+                "sort": 2,
+                "value_type": "individual"
+            },
+            "type": "graph",
+            "xaxis": {
+                "buckets": null,
+                "mode": "time",
+                "name": null,
+                "show": true,
+                "values": []
+            },
+            "yaxes": [
+                {
+                    "decimals": null,
+                    "format": "ms",
+                    "label": "",
+                    "logBase": 1,
+                    "max": null,
+                    "min": null,
+                    "show": true
+                },
+                {
+                    "format": "short",
+                    "label": null,
+                    "logBase": 1,
+                    "max": null,
+                    "min": null,
+                    "show": true
+                }
+            ],
+            "yaxis": {
+                "align": false,
+                "alignLevel": null
+            }
+        },
+        {
+            "content": "<div class=\"text-center dashboard-header\">\n  <span>INBOUND TRAFFIC BY REMOTE SERVICE</span>\n</div>",
```

Comment:
Nit: I'm not sure if it's 100% accurate to call this inbound traffic... from what I can see, the metrics on these queries are gathered on the clients calling these mirrored services, where `direction=outbound`. WDYT on calling this just "TRAFFIC BY REMOTE SERVICE"? (Also those services might have other incoming traffic not originated in the current cluster).